### ISSUE TITLE
Admin\User Profile - Users can view and edit profile

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1696]
#### Description:
As an internal and external user, given that I am a registered user in BCIERS, I have a user profile

#### Acceptance Criteria:

Given that I clicked on ~~My Profile~~ my user name in the navigation banner
Then I am brought to my profile page
[Figma link](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7854-2&t=lP4wgjHuBjopAaSN-4)

Given that I am on my profile
When I click edit
Then the fields become editable

#### Development Checklist:

- [ ] Migrate profile components from Registration to Administration app
- [ ] Change link to profile to point to `Aministration`
- [ ] Refactor profile form as per figma
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by patrickisaac** [comment]
Refinement Notes:
- @fviduya will re-test this and confirm if we can close this ticket

**Comment by fviduya** [comment]
I have tested this in the TEST environment, [here's](https://bcgov-my.sharepoint.com/:w:/g/personal/francesca_viduya_gov_bc_ca/Ea55U3u6FhtJrnwaebDTk3MBFeKPKj2L2Trguf_b4gH5ww?e=8xBGWI) the test document for reference.

I can confirm that the user can view and edit profile BUT what it doesn't do is to reflect the update on that same session. Please see the document, pages 5 and 9 shows that the name at the top right of the page updates only after logging out and logging back in. 

cc: @patriciarussellCAS / @vesselak

**Comment by fviduya** [comment]
Sure, I can create the follow-up ticket
